src/base/LMIcon/types.ts

```diff
@@ -0,0 +1,14 @@
+import { ImageStyle, ViewStyle } from "react-native";
+
+export interface LMIconProps {
+    type: 'png', // this represents the type of icon 
+    assetPath?: string, // this represents the url the icon
+    color?: string, // this represents the tintcolor of the icon
+    height?: number, // this represents the height of the icon
+    width?: number, // this represents the width of the icon
+    iconStyle?: ImageStyle, // this represents the style of the icon
+    boxFit?: 'center' | 'contain' | 'cover' | 'repeat' | 'stretch', // this defines the fit behaviour of the icon inside the box around it
+    boxStyle?: ViewStyle // this represents the style of the view
```

Comment:
Does this cover `backgroundColor`, `borderRadius`, `boxPadding` and `borderSize` all of them?

Reply:
Yes